research/flamby/fed_heart_disease/fenda/run_fold_experiment.slrm

```diff
@@ -117,6 +117,7 @@ do
     nohup python -m research.flamby.fed_heart_disease.fenda.server \
         --config_path ${SERVER_CONFIG_PATH} \
         --server_address ${SERVER_ADDRESS} \
+        --run_name ${RUN_NAME} \
```

Comment:
This doesn't appear to be used by the FENDA server in any way.

Reply:
It also looks like the fed_heart_disease FENDA server doesn't parse this argument at all, which might throw an error.